<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowWindow2</class>
 <widget class="QWidget" name="ShowWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>Выберите интересующую Вас категорию:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="ensembleButton">
     <property name="text">
      <string>Ансамбли</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QRadioButton" name="musicianButton">
     <property name="text">
      <string>Музыканты</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="countLabel">
     <property name="text">
      <string>Итого найдено пластинок: 0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Выберете интересующий Вас ансамбль:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
